### PR TITLE
Wrote policy test for full ingress restriction (no data written)

### PR DESCRIPTION
**Pull request by alxmrs**
Here is a small additional policy test.

### CODE REVIEW

File: javatests/arcs/android/integration/policy/scenarios.arcs [hunk 1]
```diff
@@ -12,6 +12,29 @@ meta
 
 import './particles.arcs'
 
+/// Scenario: No data is written
+@egressType('GeneralEgress')
```
Comment (bgogul) [inline javatests/arcs/android/integration/policy/scenarios.arcs:16]: Ingress restricting should allow the fields mentioned in the policy to be persisted. However, due to the fact that we are not plumbing the policy to `recipe2plan`, that does not happen. 

Either remove the policy or add a reference to bug b/162992168

Comment (alxmrs) [reply javatests/arcs/android/integration/policy/scenarios.arcs]: I decided to do both. How does this look?

File: javatests/arcs/android/integration/policy/scenarios.arcs [hunk 2]
```diff
@@ -12,6 +12,31 @@ meta
...
+// }
+
+// Given a recipe that is compliant with a policy
+// @policy('NoDataWrittenPolicy')
+recipe NoDataWritten
+  // And the recipe ingresses a schema Thing {a, b, c, d}
+  handle1: create
+
+  IngressThing
+    input: handle1
```
Comment (bgogul) [inline javatests/arcs/android/integration/policy/scenarios.arcs:36]: Let us call the name of the handle connection to be ingress?
```suggestion
    ingress: handle1
```

Can you also change the handle connection in the egress particle to be `egress`?

`input` and `output` confuses me. :)

File: javatests/arcs/android/integration/policy/scenarios.arcs [hunk 3]
```diff
@@ -12,6 +12,31 @@ meta
...
+//     c,
+//     d
+//   }
+// }
+
+// Given a recipe that is compliant with a policy
+// @policy('NoDataWrittenPolicy')
+recipe NoDataWritten
+  // And the recipe ingresses a schema Thing {a, b, c, d}
+  handle1: create
```
Comment (bgogul) [inline javatests/arcs/android/integration/policy/scenarios.arcs:33]: why `handle1` and not just `handle`?

Comment (alxmrs) [reply javatests/arcs/android/integration/policy/scenarios.arcs:33]: `handle` is a reserve word. Changing `handle1` causes a compiler error.

**Review by bgogul — APPROVED** [review]

**Pull request closed without merging**